### ISSUE TITLE
Charts not showing

### ISSUE THREAD

**Issue by Ahelsinger** [business-science/pytimetk#280]
I ran this code copy pasted in Pycharm but no charts were shown

https://business-science.github.io/pytimetk/tutorials/02_finance.html

`import pytimetk as tk
import pandas as pd
import numpy as np

stocks_df = tk.load_dataset("stocks_daily", parse_dates = ['date'])

stocks_df.glimpse()

# plotly engine
stocks_df.groupby('symbol').plot_timeseries(
        'date', 'adjusted',
        facet_ncol=2,
        smooth=True,
        smooth_frac=0.10,
        width=900,
        height=700,
        engine='plotly',
    )`

**Comment by girdeux31** [comment]
Same problem here. It is not obvious how to show plots. From the documentation:

`import pytimetk as tk
import pandas as pd

df = tk.load_dataset('bike_sales_sample')
df['order_date'] = pd.to_datetime(df['order_date'])

fig = df.groupby('category_1').plot_timeseries(
    date_column = 'order_date',
    value_column = 'total_price',
    smooth_frac = 0.8
)`

But plot is not shown. I also tried different engines, methods gives back a Figure object but no image is shown. I also tried with `fig.shown()` method, with no results.
